### PR TITLE
Enhance/xls testing

### PR DESCRIPTION
**Pull request by edlins**
Fixes #.

Changes proposed in this pull request:
- **XlsTestsUtils**: changed testXlsIndicator() to take T... params
- **XlsTestsUtils**: params follow Param header, data follow Date header
- **XlsTestsUtils**: data comments have "//" in the first column

- [x] added an entry to the unreleased section of `CHANGES.md`

### CODE REVIEW

**Comment by edlins** [comment]
All of this is setup for arbitrary type and number of parameters in xls validation.  I will be able to use this for LinearTransactionCost xls validation, which takes three parameters.

File: ta4j-core/src/test/java/org/ta4j/core/XlsTestsUtils.java [hunk 1]
```diff
@@ -51,83 +51,87 @@ public static void setParamValue(Sheet sheet, int paramIndex, double value) {
...
-    public static void testXlsIndicator(Class testClass, String xlsFileName, int param1Value, int valueColumnIdx, IndicatorFactory indicatorFactory) throws Exception {
-        testXlsIndicator(testClass, xlsFileName, new Double(param1Value), null, valueColumnIdx, indicatorFactory);
-    }
-
-    public static void testXlsIndicator(Class testClass, String xlsFileName, int param1Value, int param2Value, int valueColumnIdx, IndicatorFactory indicatorFactory) throws Exception {
-        testXlsIndicator(testClass, xlsFileName, new Double(param1Value), new Double(param2Value), valueColumnIdx, indicatorFactory);
+    public static <T> void testXlsIndicator(Class testClass, String xlsFileName, int valueColumnIdx, IndicatorFactory indicatorFactory, T... params) throws Exception {
+        Double[] doubleParams = new Double[params.length];
+        for (int i = 0; i < params.length; i++) {
+            doubleParams[i] = Double.valueOf(params[i].toString());
```
Comment (edlins) [inline ta4j-core/src/test/java/org/ta4j/core/XlsTestsUtils.java]: Should this be Decimal instead of Double?

Comment (team172011) [reply ta4j-core/src/test/java/org/ta4j/core/XlsTestsUtils.java]: yes

**Comment by edlins** [comment]
With this, an XLS test file has the following layout:
```
...
Param  | ... (this is the parameters header row)
param1 | value1
...
Date   | ... (this is the data header row)
date1  | open1 | high1 | low1 | close1 | volume1 | ...
//     | ... (this is a data comment)
...
```
Parameters should come before data, and there needs to be enough rows available between Param and Date to accommodate all of the T... params passed into testXlsIndicator().  Empty rows are ignored everywhere.  Data rows with "//" in the first cell are ignored as comments.

**Pull request merged**